app/services/merits_task_service.rb

```diff
@@ -40,7 +40,7 @@ def add_tasks_to_response(ccms_code)
   end
 
   def add_application_level_tasks_to_response(proceeding_type)
-    applicable_tasks = proceeding_type.application_tasks.where("proceeding_type_merits_tasks.display_rules IS NULL")
+    applicable_tasks = proceeding_type.application_tasks.where("proceeding_type_merits_tasks.display_rules IS NULL or display_rules = 'always_unless_all_da_and_non_defendant'")
```

Comment:
```suggestion
    applicable_tasks = proceeding_type.application_tasks.where("proceeding_type_merits_tasks.display_rules IS NULL OR proceeding_type_merits_tasks.display_rules = 'always_unless_all_da_and_non_defendant'")
```
OR if `proceeding_type_merits_tasks` ref not needed
```
    applicable_tasks = proceeding_type.application_tasks.where("display_rules IS NULL OR display_rules = 'always_unless_all_da_and_non_defendant'")
```

OR with ActiveRecord
```
    applicable_tasks = proceeding_type.application_tasks.where(display_rules: nil).or(display_rules: "always_unless_all_da_and_non_defendant")
```

Reply:
Updated the sql query... once the mini-loop is released this whole service is going to be removed, hopefully soon!